docker-compose.yml

```
@@ -1,24 +1,40 @@
+  openwf:
+    # build: .
+    image: ghcr.io/spaceninjaserver/SpaceNinjaServer:latest
+    environment:
+      APP_MONGODB_URL: mongodb://openwf-db:spaceninjaserver:27017/
 
-networks:
-    docker:
-        external: true
+      # Following environment variables are set to default image values.
+      # Uncomment to edit.
+      
```

yamllint

[error] trailing spaces

(trailing-spaces)